uqcsbot/cowsay.py

```diff
@@ -0,0 +1,136 @@
+
+from typing import Optional, Literal
```

Comment:
Missing List from typing
```suggestion
from typing import Optional, Literal, List
```